commonlands/player.lua

```diff
@@ -1,4 +1,7 @@
 function event_enter_zone(e)
   local client = eq.get_entity_list():GetClientByID(e.self:GetID());
-  client:MovePC(22, -144, -1543, 2, 244); -- Zone: ecommons
+  
+  if(($client->GetClientVersionBit() & 3) != 0) { #062/Titanium
```

Comment:
You are mixing perl syntax into a lua file. This will not function as is.

Reply:
Yeah, I'm out of my wheelhouse here. I'll spend some more time on it.

Based on the origination of this, do you think it's possible to remove this altogether? It looks like it was also implemented in two other places, https://github.com/ProjectEQ/projecteqquests/commit/36ebcaaf6e5f1c4e88e0de57757dcb52dcdebe17